sample-apps/async-example/raygunSetup.php

```diff
@@ -4,44 +4,34 @@
     require_once 'vendor/autoload.php';
     require_once 'config.php';
 
-    use GuzzleHttp\Client;
+    // Needed during development
+    require_once '../../src/Raygun4php/Factories/Interfaces/TransportFactoryInterface.php';
+    require_once '../../src/Raygun4php/Factories/Interfaces/HttpClientFactoryInterface.php';
+    require_once '../../src/Raygun4php/Factories/Interfaces/RaygunClientFactoryInterface.php';
+    require_once '../../src/Raygun4php/Factories/TransportFactory.php';
+    require_once '../../src/Raygun4php/Factories/HttpClientFactory.php';
+    require_once '../../src/Raygun4php/Factories/RaygunClientFactory.php';
```

Comment:
You shouldn't need to do this if you've installed the library into a project using composer, but I understand that if you're running this directly from the folder it sits in then you may do. In that case, you may want to `require_once '../../vendor/autoload.php`;` instead of doing these individually

Reply:
Thanks @robbieaverill, that seems really obvious now 🤦‍♂